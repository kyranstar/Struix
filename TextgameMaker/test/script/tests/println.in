println(5);
println("Hey");
println(true);
println(null);

print(5);
print("Hey");
print(true);
print(null);