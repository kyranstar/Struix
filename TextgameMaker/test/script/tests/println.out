5
Hey
true
NULL
5HeytrueNULL